doc/examples/ridges/plot_ridge_filter.py

```diff
@@ -0,0 +1,76 @@
+"""
```

Comment:
I'd suggest to move this file to the ``edges`` subfolder, and to change the title of this section (in ``edges/README.txt`` to "Edges, ridges and lines``.

Reply:
Ok!